### ISSUE TITLE
Add “Make an extraction” on sample index page [ACAD #6]

### ISSUE THREAD

**Issue by Li-ReDBox** [acadsamplesdb/acad_samples#27]
Samples menu -> Samples. Ability to select samples with the check box and then pick “Make an extraction” from same drop down box as [Export to CSV, Make a project]

**Comment by Li-ReDBox** [comment]
@sgoodswen: could you please confirm this is to add an extra dropdown item after "Make a project" on sample's index page?

Also, do you know if there is code handling "Make an extraction" from a list of samples?

**Comment by sgoodswen** [comment]
Yes, this enhancement is to add an extra dropdown item after "Make a project" on sample's index page?

Sorry,  I  do not know if there is code handling "Make an extraction" from a list of samples but I expect there will not be.

**Comment by Li-ReDBox** [comment]
Thank you @sgoodswen .

If "Make an extraction" is a "new" function, would you be able to explain to me how you want it to perform? I assume there could be some similar function I can learn from, but I am not sure which one.

**Comment by sgoodswen** [comment]
Li, could you please put this enhancement on hold until I find out more details from the users.

Thanks

**Comment by sgoodswen** [comment]
The workflow for this request is as follows:

Samples menu -> Samples. 

User to select samples with the check box. These samples may be on multiple pages. User then picks “Make an extraction” from same drop down box as [Export to CSV, Make a project]. 

The “New extraction” fields appear as per the [Add extraction] option. On selecting  [Submit] button, the  “ID” and “Sample” fields in the Extraction results are automatically populated with the selected samples.
